### PR TITLE
How to use small sustainable habits to get your first job

### PR DESCRIPTION
**Pull request by luojiyin1987**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by luojiyin1987** [comment]
#465

File: chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md [hunk 1]
```diff
@@ -1,160 +1,160 @@
...
+信息的数量不是问题。我们可以获取比以往更多的信息。但问题是如何有效利用我们获取的信息。
 
-The more I progress in my life and career, the more convinced I am that the path to success does not lie in setting massive goals, and then cranking every day to reach them.
+我在生活和事业获得的收获越大，我越相信，通往成功的道路不是设定巨大的目标，然后每天都为实现这些目标而努力。
 
-The path lies in choosing your goals, then designing simple systems built with small habits that will facilitate achieving those goals.
+路径在于你选择的目标，然后设计简单的系统，用于小习惯促进实现这些目标。
 
-But one crucial component is to not measure your success when you reach your goal, but measure your success each time you successfully execute on your system.
+但是一个关键的部分，不要在达到目标时衡量你的成功，而是在每次成功执行在你的系统内。
```
Comment (miyaliu666) [inline chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md:16]: 而是以每次成功执行系统中的一个任务来衡量你的成功

File: chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md [hunk 2]
```diff
@@ -1,160 +1,160 @@
...
+让我们开始吧.
 
-## Overview of the Tiny Habits Method
+## 小习惯的的概述
 
-Very briefly, let's go over the philosophy and methodology of Tiny Habits. At its core, the system follows the idea that the best way to make consistent, lasting change is to make that change tiny – embarrassingly so.
+非常简单，让我们看看小习惯的理念和方法，在其核心部分，该系统遵循的理念是，最佳方法是做出持续的小改变。
 
-Fogg argues that motivation is a poor tool to rely on when making lasting change. It fools us into thinking we can make significant change, because significant change is possible for a short time when we feel motivated.
+Fogg 认为在进行持续的改变时，动力是可怜的工具。它让我们认为我们可以在短时间内发生重大改变。
```
Comment (miyaliu666) [inline chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md:38]: 动力是糟糕的工具

File: chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md [hunk 3]
```diff
@@ -1,160 +1,160 @@
...
+## 需要关注的事
 
-We want to focus on three main components for building out our dev career:
+我们想把重点放在建立我们的开发工作的三个主要组成部分:
 
-1.  Develop your skills
-2.  Establish an online presence
-3.  Start networking
+1.  你的开发技能
+2.  建立一个线上的东西
```
Comment (miyaliu666) [inline chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md:81]: 建立在线形象

File: chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md [hunk 4]
```diff
@@ -1,160 +1,160 @@
...
+3.  网络影响力
```
Comment (miyaliu666) [inline chinese/articles/how-to-use-small-sustainable-habits-to-get-your-first-dev-job.md:82]: 开始社交

**Pull request merged**